scripts/esaview.py

```diff
@@ -87,26 +87,28 @@ def get_args():
     parser = argparse.ArgumentParser(description='Run Earth Solid Angle viewer')
     parser.add_argument('infile', type=str,
                        help='Input data file root (e.g. FEB1411)')
+    parser.add_argument('--data-root', type=str, default='$SKA/data/acis_taco',
```

Comment:
This path still needs a fix (noticed in meeting with Tom and I think he took an action to do the fix, but figured I'd comment before I forgot about it).